src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp

```diff
@@ -241,7 +246,37 @@ struct EvolutionMetavars {
         intrp::callbacks::ObserveTimeSeriesOnSurface<tags_to_observe, AhB, AhB>;
   };
 
-  using interpolation_target_tags = tmpl::list<AhA,AhB>;
+  struct ControlTest
```

Comment:
Would it be helpful to add a comment like `This is just used to test that the control system is invoked. In the future this will be updated with actually controlling the maps.` so that others will have some context when looking at this file? E.g. I'm doing some observer related changes and if I get a conflict while rebasing that tidbit of info would probably help me :)

Reply:
Nah that won't cause any trouble. In my commits I add the necessary aliases and such already.